src/Components/button.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+
 const Button = props => {
   const { name, size, last } = props;
   if (name) {
```

Comment:
Also I am curious why would a button not have name? and do you really want to return null, what gets rendered then? :thinking: